Sources/TuistKit/Extensions/Generator+TuistKit.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import TuistGenerator
+
+extension Generator {
```

Comment:
I created this extension to initialize the generator from `TuistKit` without having to inject the dependencies.